BE/PoroDev.Runtime/Extensions/DockerImageService.cs

```diff
@@ -11,6 +14,132 @@ public DockerImageService() : base()
 
         }
 
+        public async Task<List<string>> CheckAndCreateDockerfile(List<string> argumentList, IZipManipulator _zipManipulator)
```

Comment:
Could you explain why was IZipManipulator injected like this and not through DI?